cnf-certification-test/lifecycle/suite.go

```diff
@@ -148,6 +148,13 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.GuaranteedPods)
 		testCPUIsolation(&env)
 	})
+
+	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestAffinityRequired)
+	ginkgo.It(testID, ginkgo.Label(tags...), func() {
```

Comment:
I think this workflow is not working as expected, I'll show you an example in the comment including these minor comments.

Reply:
Ah yes, it's because of the multiple `It` Ginkgo blocks.  I'll fix.